assignments/arrays.js

```diff
@@ -63,34 +63,61 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+console.log(`Car 33 is a ${inventory[32].car_year} ${inventory[32].car_make} ${inventory[32].car_model}`);
 
 
 
 // ==== Challenge 2 ====
-// The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+// The dealer needs the information on the last car in their inventory.  
+// What is the make and model of the last car in the inventory? Log the make and model into the console.
+let lastCar = inventory[49];
```

Comment:
While this works, I think the idea that we wanted you to show here is something along the lines of:
```let lastCar = (inventory.length -1)```
We wont always know the length of the array that we are trying to get data from so doing it this way is more explicit and will work every time. There could be cases were if something got pushed later into the inventory array, this would no longer work.